### PR TITLE
fix: include accept header in fetchHeader

### PR DESCRIPTION
**Pull request by dylandepass**
fixes: #264

Could include this as well as the browser does.
`*/*;q=0.8`

### CODE REVIEW

File: src/MediaHandler.js [hunk 1]
```diff
@@ -357,6 +357,7 @@ export default class MediaHandler {
       headers: {
         range: 'bytes=0-8192',
         'accept-encoding': 'identity',
+        accept: 'image/jpeg,image/jpg,image/png,image/gif,video/mp4,application/xml,image/x-icon',
```
Comment (tripodsan) [inline src/MediaHandler.js:360]: why application/xml ? is this for svg?

**Comment by tripodsan** [comment]
> Could include this as well as the browser does.
> `*/*;q=0.8`

I tihnk this would be good for backward compat.

**Comment by tripodsan** [comment]
we should run some word2md and gdocs2md imports with the changed media handler to be sure the major use cases are not compromised.

File: src/MediaHandler.js (hunk 1 above)
Comment (dylandepass) [reply src/MediaHandler.js:360]: Yeah wasn't 100% sure if that is needed. Pulled it from [here](https://github.com/adobe/helix-admin/blob/6af10e82cd4de6a5d8816fb0a587b6c7f302f9d5/src/preview/media-upload.js#L133).

**Comment by dylandepass** [comment]
> we should run some word2md and gdocs2md imports with the changed media handler to be sure the major use cases are not compromised.

After looking closer I believe this code path is only encounted in html2md since the image blobs are contained within the document when `word2md` and `gdocs2md` is run and nothing is downloaded directly. Unless I'm missing other situations where image links are downloaded from word docs?

Going to add @dominique-pfister to the review just to get another set of eyes on this in case I'm missing something else.

**Review by dominique-pfister — APPROVED** [review]

**Comment by tripodsan** [comment]
> only encounted in html2md

yes. of course.

**Review by tripodsan — APPROVED** [review]

**Pull request merged**